### ISSUE TITLE
Faster transaction inserting: Automatically append the first default currency

### ISSUE THREAD

**Issue by pmarciniak** [beancount/fava#1144]
For folks like me, who use the manual transaction inserting tool a lot, it would be a great time saver if one didn't have to explicitly enter the default currency on each posting.

Currently, when I simply enter "2+3" in the Amount box, I'll get "5" in my beancount file. While this is valid beancount syntax, it's not very elegant. Ideally, I'd like my Amount entry to be automatically converted to "5.00 USD" where USD is the first of my operating currencies.

**Comment by pmarciniak** [comment]
Even better idea: if the account in question only has a single currency, populate the currency based on the account. That way currencies can be auto-populated not only for account in the main currency, but also for "foreign" currency accounts.

**Comment by yagebu** [comment]
There's already the autocomplete which suggests your most common currency, so I don't think this is needed.

**Comment by fdw** [comment]
For me, autocompletion only suggests currencies, so I have to press at least two more keys to add the currency. This proposal would need no additional key presses (besides the amount).

Can I set autocompletion to actually type the currency for me, without additional work?

**Comment by yagebu** [comment]
Yes, maybe this could be quickened by more aggressively autocompleting the currency. Using the `selectFirst` prop for the Autocomplete Svelte component could do the trick.

**Comment by yagebu** [comment]
Implemented in #2331, the currency is now automatically selected (and only account currencies are suggested)
